cishouseholds/pipeline/regex_patterns.py

```diff
@@ -3,6 +3,25 @@
 """
 from collections import namedtuple
 
+# list of common job titles - these are meant to be used as negative patterns for "at school",
+# "in college" or "attending university"
+occupations = [
+    "ASSISTANT",
+    "CATERING",
+    "CHIEF",
+    "INTERN",
+    "INVIGILATOR",
+    "LECTURER",
+    "MANAGER",
+    "MASTER",
+    "MINDER",
+    "MISTRESS",
+    "PROFESSOR",
+    "SUPERVISE",
+    "TEACH(ER|ING)?",
+    "WORKER",
+]
+
```

Comment:
Could we add: 'SECRETARY', 'MANAGER/MANAGEMENT' , 'ADMINISTRAT[OR/ION]', 'DIRECTOR', 'DRIVER', 'COOK', 'CHEF', 'COACH', 'CONSULTANT', variations on spelling of professor (proffessor, profesor)... in spite of them being Professors.... :)